### DISCUSSION TITLE
Enabling dev mode in the extensions menu

### DISCUSSION THREAD

**Discussion by Haxnstuff** [3kh0/ext-remover#159] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Is there a way to force dev mode?
![image](https://user-images.githubusercontent.com/76791182/198636297-9ff3169d-b08b-4b39-8395-66fe0f34e7f4.png)

**Comment by ghost** [comment]
I'm not really sure. If there was, it would be similar to how the exploit worked, but it seems as they patched that too (plus, it's going to have to be overhauled to work)

**Comment by Haxnstuff** [comment]
damn, well thanks for telling me

**Reply by ghost** [reply]
you're very welcome
